src/main/java/address/model/UserPrefs.java

```diff
@@ -35,4 +40,12 @@ public String getSaveLocationString() {
     public boolean isSaveLocationSet() {
         return getSaveLocation() != null;
     }
+
+    public ScreenSize getScreenSize() {
+        return screenSize == null ? ScreenUtil.getRecommendedScreenSize() : screenSize;
```

Comment:
`return screenSzie == null ? new ScreenSize() : screenSize;`
The default constructor will use default values.